specs/templates/custom-component.md

```diff
@@ -65,13 +65,19 @@
 
 *Work closely with the visual design partner to co-develop the API and anatomy along side the visual design.*
 
+### Native form integration
+
+*Describe the plan for custom element form integration or why it's not necessary.*
+
```

Comment:
There should probably be a link in here and discussion about aligning with aria-patterns: https://www.w3.org/WAI/ARIA/apg/patterns/

And falling back to research of aria patterns in the community and  checking open issues in the aria repo

Reply:
oh wait, you have that below lol